.eslintrc

```diff
@@ -7,7 +7,7 @@
     "jest": true
   },
   "rules": {
-    "@typescript-eslint/ban-ts-ignore": "warn",
+    "@typescript-eslint/ban-ts-comment": "warn",
```

Comment:
Why you changed this eslint rule?

Reply:
I was getting this error:
<img width="850" alt="Screen Shot 2020-08-19 at 11 36 51" src="https://user-images.githubusercontent.com/19983991/90648733-4993d280-e210-11ea-9edf-f486ef049489.png">
And [here](https://github.com/typescript-eslint/typescript-eslint/issues/2286#issuecomment-656629773) they say it was changed to "comment"